Gemfile.lock

```diff
@@ -202,13 +202,19 @@ GEM
     guard-rspec (4.2.8)
       guard (~> 2.1)
       rspec (>= 2.14, < 4.0)
-    haml (4.0.5)
+    haml (4.0.7)
       tilt
     haml-rails (0.5.3)
       actionpack (>= 4.0.1)
       activesupport (>= 4.0.1)
       haml (>= 3.1, < 5.0)
       railties (>= 4.0.1)
+    haml_lint (0.24.0)
+      haml (>= 4.0, < 5.1)
+      rainbow
+      rake (>= 10, < 13)
+      rubocop (>= 0.36.0)
```

Comment:
did you manually modified this? 

Because haml_lint 0.24.0 depends on Rubocop 0.47.0 https://rubygems.org/gems/haml_lint/versions/0.24.0